### PR TITLE
Added a flag to enable human input for Bedrock Agents

### PR DESCRIPTION
**Pull request by jdbaker01**
- Simplify the create of an agent which uses human input by including a flag (False by default to match current behavior). 
- Add new integration test for the use case.

### CODE REVIEW

**Comment by jdbaker01** [comment]
Will update, fixing lint.

**Review by 3coins — COMMENTED** [review]
@jdbaker01 
This seems a bit different from the human confirmation input that your referenced in the [documentation](https://docs.aws.amazon.com/bedrock/latest/userguide/agents-userconfirmation.html). The test sample you have included seems purely prompt based. Should the CONFIRM or DENY from the agent response be handled anywhere in the code?

File: libs/aws/tests/integration_tests/agents/test_bedrock_agents.py [hunk 1]
```diff
@@ -435,3 +435,67 @@ def should_continue(data):
...
+        agent = BedrockAgentsRunnable.create_agent(
+            agent_name="weather_agent",
+            agent_resource_role_arn=agent_resource_role_arn,
+            foundation_model=foundation_model,
+            instruction="""
+                You are an agent who helps with getting weather for a given location.
+                If the user does not provide a location then ask for the location and be
+                sure to use the word 'location'.""",
+            tools=tools,
+            enable_humidity=True,
```
[the remark this answers is not in the document]
Comment (jdbaker01) [reply libs/aws/tests/integration_tests/agents/test_bedrock_agents.py:489]: I have no idea how that happened, lol. I will update tomorrow and post new. Maybe spell checker?!?

**Comment by jdbaker01** [comment]
> @jdbaker01 This seems a bit different from the human confirmation input that your referenced in the [documentation](https://docs.aws.amazon.com/bedrock/latest/userguide/agents-userconfirmation.html). The test sample you have included seems purely prompt based. Should the CONFIRM or DENY from the agent response be handled anywhere in the code?

That is a different feature, this enables human input if the agent doesn’t have all the parameters. The confirm/deny is a separate issue and is “human confirmation”. Confirmations can be added to individual  action whereas human input is applicable to all actions and is not confirm/deny but asks for missing params.

**Review by 3coins — APPROVED** [review]
@jdbaker01 
Looks good!

**Pull request merged**